capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -8,12 +8,11 @@ const addBookToBookList = (bookId, bookListId) => {
 }
 
 const BookSearchTile = (props) => {
-  // TODO: #48 Prompt user to create a BookList if they have no BookLists
   return (
     <div className="book-search-tile">
       <Container>
         <Row className="justify-content-md-center">
-          <Col md="auto"><img className="book-img-med" src={props.book.thumbnailLink} alt={props.book.title}/></Col>
+          <Col md="auto"><img className="book-img-med" src={props.book.thumbnailLink} alt={props.book.title} /></Col>
```

Comment:
nit: You should be able to omit the closing `</Col>` tag since the tag is self-closing with the `/` at the end. Agree with adding the extra space here though.

Reply:
When I remove the closing Col tag, it tells me there is a JSX syntax error. Could this be because the img tag is a child of this object so it's necessary?

Reply:
My mistake! Totally missed the opening `<img` tag so this makes much more sense now. I'd move the `img` onto its own indented line so it's harder to miss but otherwise this looks good.